app/api-client.js

```diff
@@ -24,14 +24,16 @@ class JWTAuth {
   }
 }
 
+exports.JWTAuth = JWTAuth;
+
 
 class APIClient {
   constructor(conf) {
     this.base_url = conf.base_url;
     this.auth = null;
   }
 
-  request(endpoint, method = 'GET', body = null) {
+  request(endpoint, { method='GET', body=null, params={} } = {}) {
```

Comment:
This is odd looking, but it is the only way I found to get named function parameters with default values.

Reply:
Maybe I'm missing something here, could this be tackled by having something like?

```javascript
  request(endpoint, args = {}) {
    const method = args.method || 'GET';
    const params = args.params || {};
    const body = args.body || null;
    // ...
```

Obviously not a big deal, it's just a bit odd looking as you said (is this restructuring here?).

Reply:
That code is equivalent, but way more verbose - the method I'm using is correct, I just thought it might raise a question or two.

Reply:
Ok, so it's not the _only_ way.